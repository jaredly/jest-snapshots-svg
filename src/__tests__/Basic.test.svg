<?xml version="1.0" encoding="UTF-8" ?>
<svg width="320" height="480" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">


 <rect type="View" x="0.5" y="0.5" width="319" height="479" fill="none" stroke="red" stroke-width="1"/>

 <g transform='translate(0, 0)'>
   <image xlink:href="example.png" preserveAspectRatio="" x="136" y="25" width="48" height="48"/>
   <image xlink:href="https://cdn.kastatic.org/images/badges/meteorite/ten-to-fourth-40x40.png" preserveAspectRatio="" x="135" y="73" width="50" height="50"/>
   <image xlink:href="example.png" preserveAspectRatio="" x="136" y="123" width="48" height="48"/>
   <text><tspan x="127" y="183.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">Hello folks</tspan></text>
   <rect type="View" x="135" y="189" width="50" height="50" fill="powderblue"/>
   <text><tspan x="112" y="251.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">Ok </tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="bold" font-style="normal" font-size="14">here</tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14"> is</tspan>
<tspan x="112" y="269.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">something</tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14"> </tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="italic" font-size="14">long</tspan>
<tspan x="112" y="287.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="italic" font-size="14">that </tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">is going to</tspan>
<tspan x="112" y="305.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">wrap for me.</tspan></text>
   <rect type="View" x="135" y="311" width="50" height="50" fill="skyblue"/>
   <path type="View" fill="steelblue" d="M135,361L185,361A0,0 0 0,1 185,361L185,411A0,0 0 0,1 185,411L135,411A0,0 0 0,1 135,411L135,361A0,0 0 0,1 135,361Z"/><path fill="yellow" d="M135,361L185,361A0,0 0 0,1 185,361L185,411A0,0 0 0,1 185,411L135,411A0,0 0 0,1 135,411L135,361A0,0 0 0,1 135,361ZM135,361L135,411A0,0 0 0,0 135,411L185,411A0,0 0 0,0 185,411L185,361A0,0 0 0,0 185,361L135,361A0,0 0 0,0 135,361Z"/>
   <rect type="View" x="138" y="411" width="44" height="44" fill="green" rx="10" ry="10"/>

   <g transform='translate(138, 411)'>
     <rect type="View" x="11" y="11" width="22" height="22" fill="blue" stroke="orange" stroke-width="2" rx="11" ry="11"/>
   </g>

 </g>

</svg>
