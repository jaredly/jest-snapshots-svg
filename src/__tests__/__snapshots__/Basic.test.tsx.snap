// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counting nodes it is good with memory 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "red",
      "borderWidth": 1,
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "src/__tests__/example.png",
      }
    }
  />
  <Image
    height={50}
    source={
      Object {
        "uri": "https://cdn.kastatic.org/images/badges/meteorite/ten-to-fourth-40x40.png",
      }
    }
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
    width={50}
  />
  <Image
    source={
      Object {
        "testUri": "src/__tests__/example.png",
      }
    }
    style={
      Object {
        "tintColor": "red",
      }
    }
  />
  <Text>
    Hello folks
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "powderblue",
        "height": 50,
        "width": 50,
      }
    }
  />
  <Text
    style={
      Object {
        "maxWidth": 100,
      }
    }
  >
    Ok 
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      here
    </Text>
     is something
     
    <Text
      style={
        Object {
          "fontStyle": "italic",
        }
      }
    >
      long that 
    </Text>
    is going to wrap for me.
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "skyblue",
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "steelblue",
        "borderBottomWidth": 10,
        "borderColor": "yellow",
        "borderWidth": 2,
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "borderRadius": 10,
        "padding": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "blue",
          "borderColor": "orange",
          "borderRadius": 100,
          "borderWidth": 2,
          "padding": 10,
        }
      }
    />
  </View>
</View>
`;
